tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -160,13 +207,14 @@ func TestMain(m *testing.M) {
 }
 
 func TestGlobalCheckAndReport(t *testing.T) {
-	if err := visitProductPage(testRetryTimes); err != nil {
+	if err := visitProductPage(productPageTimeout, http.StatusOK); err != nil {
 		t.Fatalf("Test app setup failure: %v", err)
 	}
 
 	glog.Info("Successfully sent request(s) to /productpage; checking metrics...")
 	// must sleep to allow for prometheus scraping, etc.
 	time.Sleep(15 * time.Second)
+	dumpMixerMetrics()
```

Comment:
i don't think this offers anything beyond the promDump() changes in the other outstanding PR. I don't think there is any evidence that the mixer->prometheus parts of this test are flaky or problematic.

Reply:
sure I can get rid of this. I needed something for my debugging.